app/controllers/user_sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class UserSessionsController < ApplicationController
+  skip_before_filter :require_login, except: [:destroy]
+  
+  def new
+    @user = User.new
+  end
+
+  def create
+    if @user = login(params[:email], params[:password])
+      redirect_back_or_to(root_path, notice: 'Login successful')
+    else
+      flash.now[:alert] = 'Login failed'
+      render action: 'new'
+    end
+  end
+
+  def destroy
+    logout
+    redirect_to(root_path, notice: 'Logged out!')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.